dozer/cogs/modmail.py

```diff
@@ -149,6 +157,17 @@ async def configure_modmail(self, ctx: DozerContext, target_channel):
         await config.update_or_add()
         await ctx.reply("Configuration saved!")
 
+    @command()
+    @has_permissions(manage_roles=True)
+    async def start_modmail_with_user(self, ctx: DozerContext, member: discord.Member):
+        """Start modmail with a user, should be used in channel with modmail button"""
+        target_record = await ModmailConfig.get_by(guild_id=ctx.interaction.guild_id)
```

Comment:
![image](https://github.com/FRCDiscord/Dozer/assets/17951807/f007490c-912d-41b6-9df5-de3d3d219b01)

This doesn't work outside of slash commands. Otherwise, looks good to me!